tests/output-eve-ftp/test.yaml

```diff
@@ -0,0 +1,65 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
```

Comment:
we will need a check for the version as well, as 4.1.x will fail this test

Reply:
Will add a `files` requirement for `src/output-json-ftp.c` which was added for FTP logging.